Dockerfile

```diff
@@ -9,17 +9,16 @@ WORKDIR /build-ocrd
 COPY setup.py .
 COPY requirements.txt .
 COPY requirements_test.txt .
+COPY ocrd_tesserocr ./ocrd_tesserocr
+COPY Makefile .
 RUN apt-get update && \
     apt-get -y install --no-install-recommends \
-    ca-certificates \
-    make \
-    git
-COPY Makefile .
-RUN make deps-ubuntu
-COPY ocrd_tesserocr ./ocrd_tesserocr
+    libtesseract-dev \
+    libleptonica-dev \
```

Comment:
`libleptonica-dev` is redundant (it is already a dependency of `libtesseract-dev`).

Reply:
Thanks, will remove.